src/Symfony/Bridge/Doctrine/Profiler/DoctrineDataCollector.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Doctrine\Profiler;
+
+use Doctrine\Common\Persistence\ManagerRegistry;
+use Doctrine\DBAL\Logging\DebugStack;
+use Doctrine\DBAL\Types\Type;
+use Symfony\Component\Profiler\DataCollector\AbstractDataCollector;
+use Symfony\Component\Profiler\DataCollector\RuntimeDataCollectorInterface;
+
+/**
+ * DoctrineDataCollector.
+ *
+ * @author Fabien Potencier <fabien@symfony.com>
+ */
+class DoctrineDataCollector extends AbstractDataCollector implements RuntimeDataCollectorInterface
+{
+    private $registry;
+    private $connections;
+    private $managers;
+    private $loggers = array();
+
+    public function __construct(ManagerRegistry $registry)
+    {
+        $this->registry = $registry;
+        $this->connections = $registry->getConnectionNames();
+        $this->managers = $registry->getManagerNames();
```

Comment:
For the Doctrine bridge, the new collector might be a good time to move the extra data collected in DoctrineBundle to the bridge itself btw.